core/components/atoms/avatarGroup/AvatarGroup.tsx

```diff
@@ -96,12 +103,16 @@ export const AvatarGroup = (props: AvatarGroupProps) => {
   const hiddenAvatarCount = list.length > max ? Math.min(list.length - max, 99) : 0;
 
   const style = {
-    borderRadius: '50%',
     backgroundColor: `${borderColor}`,
-    border: `var(--spacing-xs) solid ${borderColor}`,
-    boxShadow: `0 0 0 var(--spacing-xs) ${borderColor}`,
+    boxShadow: `0 0 0  calc(var(--spacing-xs) + var(--spacing-s)) ${borderColor ? borderColor : ''}`,
   };
 
+  const tinySize = {
```

Comment:
Please change the variable name `tinySize`

Reply:
done